pkg/server/server.go

```diff
@@ -44,6 +47,9 @@ const (
 	defaultListObjectsMaxResults            = 1000
 	defaultMaxConcurrentReadsForCheck       = math.MaxUint32
 	defaultMaxConcurrentReadsForListObjects = math.MaxUint32
+	defaultCheckQueryCacheLimit             = 10000
+	defaultCheckQueryCacheTTL               = 10 * time.Second
+	defaultCheckQueryCacheEnable            = true
```

Comment:
Default to false for the reasons mentioned earlier.

https://github.com/openfga/openfga/pull/891/files#r1284870129